res/controllers/engine-api.d.ts

```diff
@@ -301,4 +301,15 @@ declare namespace engine {
      *               SoftStart with low factors would take a while until sound is audible. [default = 1.0]
      */
     function softStart(deck: number, activate: boolean, factor?: number): void;
+
+    /**
+     * Converts a string into another charset.
+     *
+     * This function is useful to display text on a device that does not make use of UTF-8.
+     * Available charset names are listed here: http://www.iana.org/assignments/character-sets/character-sets.xhtml
+     * @param {string} targetCharset The charset to encode the string into.
+     * @param {string} value The string to encode
+     * @returns {ArrayBuffer | undefined}The converted String as an array of bytes or undefined if an error happened when performing conversion
```

Comment:
Maybe `jsdoc/require-param-type` and `jsdoc/require-returns-type` should be disabled since we're docmenting tpyes using JS.